packages/autorest.python/tasks.py

```diff
@@ -253,7 +254,8 @@ def _build_flags(
         "namespace": namespace,
         "client-side-validation": client_side_validation,
         "black": True,
-        "generate-sample": swagger_group == _SwaggerGroup.AZURE or swagger_group == _SwaggerGroup.AZURE_ARM,
+        "generate-sample": generate_sample,
+        "include-x-ms-examples-original-file": generate_sample,
```

Comment:
what does this flag do? we don't seem to use it

Reply:
@iscai-msft `include-x-ms-examples-original-file` is necessary otherwise we can't get original sample name and can only use `sample.json` as sample name.